src/env.rs

```diff
@@ -40,10 +39,10 @@ const INSTALLATION_ID_STORAGE_KEY: &str = "installation_id";
 
 #[wasm_bindgen]
 extern "C" {
-    #[wasm_bindgen(catch, js_namespace = ["self"])]
-    static app_version: String;
-    #[wasm_bindgen(catch, js_namespace = ["self"])]
-    static shell_version: Option<String>;
+    #[wasm_bindgen(js_namespace = ["self"])]
+    static APP_VERSION: String;
```

Comment:
Why rename it ?
This is used to set the app and shell version in the worker script
https://github.com/Stremio/stremio-core-web/blob/development/src/worker.js#L11

Reply:
We could override the linting but in general Statics and constants should be all capital letters.

Reply:
Those declarations will be exposed to the self namespace, meaning that it will be accessible from a JS context
The worker script is supposed to set those `app_version` and `shell_version` for the core to access them
So when the `analytics_context()` is called, it will panic because it can't access them

Reply:
Fixed!